testnet/stacks-node/src/config.rs

```diff
@@ -2389,6 +2389,7 @@ pub struct ConnectionOptionsFile {
     pub antientropy_public: Option<bool>,
     pub private_neighbors: Option<bool>,
     pub block_proposal_token: Option<String>,
+    pub antientropy_retry: Option<u64>,
```

Comment:
are there other files that need to be changed as well? i..e https://github.com/stacks-network/stacks-core/blob/chore/configure-antientropy-retry/stackslib/src/net/connection.rs#L482